docs/reference/eql/syntax.asciidoc

```diff
@@ -410,6 +410,13 @@ double quote (`"`), must be escaped with a preceding backslash (`\`).
 |`\"`             | Double quote (`"`)
 |====
 
+You can escape Unicode characters using a case-insensitive hexadecimal
+`\u{XXXXXXXX}` escape sequence. Hexadecimal values shorter than 8 characters are
```

Comment:
Could you please add a sentence that 2-8, or that max 8 hex digits  are allowed?
`values shorter than 8 chars` can imply that more than 8 are allowed.

Reply:
Thanks for the feedback @matriv. I've noted that hex values are 2-8 chars with 9b36d89713e9d225a0987f016b2f27fde6b77500.